api_server.py

```diff
@@ -202,9 +210,22 @@ def on_get(self, req, resp, client, start_slot, end_slot=None):
 
 classifier = None
 if not DISABLE_CLASSIFIER:
-    print("Initialising classifier, this could take a moment...")
-    classifier = MultiClassifier(DATA_DIR) if not DISABLE_CLASSIFIER else None
-    print("Done")
+    if MODEL_PATH != "":
+        if MODEL_PATH.endswith(".pkl"):
+            classifier = import_classifier(MODEL_PATH)
+
+        else:
+            print("model path must end with .pkl")
+            exit(0)
```

Comment:
We should use a non-zero exit code for errors

```suggestion
            exit(1)
```